documentation/assemblies/assembly-upgrading.adoc

```diff
@@ -16,7 +16,7 @@ Upgrading {ProductName} is accomplished by applying the YAML files for the new v
 ifeval::["{cmdcli}" == "oc"]
 include::../modules/proc-upgrading-enmasse-using-bundle.adoc[leveloffset=+1]
 include::../modules/proc-upgrading-enmasse-using-ansible.adoc[leveloffset=+1]
-include::../modules/proc-upgrading-router-pods.adoc[leveloffset=+1]
+// include::../modules/proc-upgrading-router-pods.adoc[leveloffset=+1]
```

Comment:
We should probably remove the module as well?

Reply:
@lulf, If you don't think we'll ever need it again, I will remove it. Otherwise, just figured it won't hurt for it to exist? 
Also, about to push an update to completely remove this line altogether.

Reply:
That is why we have git, it allows us to resurrect it later on, as long as we use a descriptive commit message :)